outputs.tf

```diff
@@ -1,31 +1,28 @@
-output "cluster" {
-  value = google_container_cluster.engineering.name
-}
-
-output "host" {
-  value     = google_container_cluster.engineering.endpoint
-  sensitive = true
-}
-
-output "cluster_ca_certificate" {
-  value     = base64decode(google_container_cluster.engineering.master_auth.0.cluster_ca_certificate)
-  sensitive = true
+output "aws_region" {
+  description = "AWS region for all resources"
+  value       = var.aws_region
 }
 
-output "username" {
-  value     = google_container_cluster.engineering.master_auth.0.username
-  sensitive = true
-}
-
-output "password" {
-  value     = google_container_cluster.engineering.master_auth.0.password
-  sensitive = true
+output "cluster" {
+  description = "Name of the EKS cluster"
+  value       = data.aws_eks_cluster.cluster.id
 }
 
 output "enable_consul_and_vault" {
-  value = var.enable_consul_and_vault
+  description = "Boolean flag to enable consul and vault"
+  value       = var.enable_consul_and_vault
 }
 
+# Does not require aws-iam-authenticator
 output "kubeconfig" {
-  value = data.template_file.kubeconfig.rendered
+  description = "Kubeconfig file for cluster management"
+  value       = data.template_file.kubeconfig.rendered
+  sensitive   = true
+}
+
+# Requires aws-iam-authenticator
+output "module_kubeconfig" {
```

Comment:
Same with this, do we need to export this output? It contains sensitive info that we don't rlly use downstream

Reply:
We need a kubeconfig in the [verify step](https://github.com/hashicorp/katakoda/blob/master/terraform-infrastructure-pipelines-workshop/verify-deployments.md) - I'd like to use the one generated from the module, just for simplicity's sake, but I want to make sure that installing aws-iam-authenticator on the katacoda env works first, so I have both for now.

Reply:
If you're installing the AWS CLI (and `aws-iam-authenticator`) on the katacoda env, we prob don't need to ouput the `kubeconfig`. I recommend using this aws command instead:

```
$ aws eks --region $(terraform output -raw aws_region) update-kubeconfig --name $(terraform output -raw cluster_id)
```


Otherwise, can we update the [`template_file`](https://github.com/hashicorp/learn-terraform-pipelines-k8s/blob/rln-awsify/main.tf#L112-L122) so it creates the file locally?